components/ide/code/leeway.Dockerfile

```diff
@@ -2,7 +2,8 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM node:12.18.3 AS node_installer
+# we use latest major version of Node.js distributed VS Code. (see about dialog in your local VS Code)
+FROM node:12.21.0 AS node_installer
```

Comment:
A comment explaining why we no longer use the VS Code-tested version (12.18.3) would be nice.